StripePayments/StripePayments/Source/PaymentHandler/STPPaymentHandler.swift

```diff
@@ -1355,6 +1356,21 @@ public class STPPaymentHandler: NSObject {
                     )
                 )
             }
+        case .promptpayDisplayQrCode:
+            guard
+                let returnURL = URL(string: currentAction.returnURLString ?? ""),
+                let presentingVC = currentAction.authenticationContext
+                    as? PaymentSheetAuthenticationContext,
+                let hostedInstructionsURL = authenticationAction.promptPayDisplayQrCode?.hostedInstructionsURL
+
+            else {
+                fatalError()
```

Comment:
Can we use an `assertionFailure` and some graceful fallback instead? I don't like to use `fatalError`s

Reply:
Hmm, fatalError usage is quite common in `STPPaymentHandler` vs `assertionFailure`. I'm not sure what a graceful fallback would look like either, any ideas?

Reply:
If we're already using it, then let's just keep it.
Graceful fallback IMO would be just returning with an error, but if we're not setup for it we shouldn't be doing it just for this